src/xcode/ENA/ENA/Source/Scenes/RiskLegend/RiskLegendViewController.swift

```diff
@@ -61,10 +61,7 @@ class RiskLegendViewController: DynamicTableViewController {
 
 	private func setupTableView() {
 		tableView.separatorStyle = .none
-		tableView.register(
-			UINib(nibName: String(describing: RiskLegendDotBodyCell.self), bundle: nil),
-			forCellReuseIdentifier: RiskLegendViewController.CellReuseIdentifier.dotBody.rawValue
-		)
+		tableView.register(RiskLegendDotBodyCell.self, forCellReuseIdentifier: RiskLegendViewController.CellReuseIdentifier.dotBody.rawValue)
```

Comment:
please keep the formatting

Reply:
@KaiTeuber I did it like this because that is the way it is mostly done throughout the project.

Reply:
Down here in the readme section is a link to a code snipet https://github.com/corona-warn-app/cwa-app-ios